temporal-server/config/dynamicconfig/development-cass.yaml

```diff
@@ -0,0 +1,42 @@
+#frontend.enableClientVersionCheck:
```

Comment:
Any reason you keep this commented out section?

Reply:
No specific reason, I fetched these files as is from the Temporal repo.